presto-main/src/test/java/com/facebook/presto/sql/query/TestFilteredAggregations.java

```diff
@@ -126,28 +179,42 @@ public void rewriteAddFilterWithMultipleFilters()
                                 tableScan(
                                         "orders", ImmutableMap.of("totalprice", "totalprice",
                                                 "custkey", "custkey")))));
+
+        assertPlan(
+                "SELECT sum(IF(totalprice > 0, totalprice)), sum(IF(custkey > 0, custkey)) FROM orders",
+                anyTree(
+                        filter(
+                                "(\"totalprice\" > 0E0 OR \"custkey\" > BIGINT '0')",
+                                tableScan(
+                                        "orders", ImmutableMap.of("totalprice", "totalprice",
+                                                "custkey", "custkey")))));
```

Comment:
make sure for each function:
- either we put all params into one line or
- make one param per line and leave the first line empty